pylintrc

```diff
@@ -81,7 +81,7 @@ confidence=
 disable=missing-docstring, bad-builtin, abstract-class-little-used, star-args,
         too-many-ancestors, too-few-public-methods, fixme, too-many-instance-attributes,
         locally-disabled, too-many-arguments, invalid-name, duplicate-code,
-        broad-except
+        broad-except, abstract-method, arguments-differ, unused-argument, no-self-use

```

Comment:
Is not a very good idea to ignore `arguments-differ`, `unused-argument` and `no-self-use`.